metricbeat/module/rabbitmq/_meta/config.reference.yml

```diff
@@ -0,0 +1,8 @@
+- module: rabbitmq
+  metricsets: ["node", "queue", "connection"]
+  enabled: true
+  period: 10s
+  hosts: ["localhost:15672"]
+
+  username: guest
```

Comment:
Do we have a  default username and password? If not we should comment these two out.

Reply:
Commented out, they are indeed not set by default.